tasks/fh-build.js

```diff
@@ -43,7 +42,7 @@ module.exports = function(grunt) {
    * If grunt config contains an array property called 'fhignore',
    * its elements will be excluded from the tarball.
    */
-  var mergePatterns = function(bundle_deps) {
+  var mergePatterns = function (bundle_deps) {
```

Comment:
Are we suggesting this change everywhere? I think that might cause a lot of diff churn in PRs. WDYT?